lib/src/konsistTest/kotlin/com/lemonappdev/konsist/api/ApiKonsistTest.kt

```diff
@@ -136,6 +135,7 @@ class ApiKonsistTest {
         }
     }
 
```

Comment:
I create ticket to simplify below methods: https://lemonappdev.atlassian.net/browse/KON-497